### PR TITLE
feat(Mobile): adds ANR/App Hangs to RN+Flutter Features

### PR DESCRIPTION
**Pull request by kahest**
## DESCRIBE YOUR PR
Adds ANR and App Hangs support to Flutter and React Native feature pages. Also refactors links to use <Link> and <PlatformLink> components and fixes some smaller errors/issues.

fixes https://github.com/getsentry/sentry-docs/issues/5771

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/platforms/android/features/index.mdx [hunk 1]
```diff
@@ -13,42 +13,42 @@ a set of experimental features to give you more details about app start performa
 
 We released <PlatformLink to="/enhance-errors/source-context">Source Context</PlatformLink>, which shows snippets of code around the location of stack frames.
 
-Let us know if you have feedback through [GitHub issues](https://github.com/getsentry/sentry-java/issues/new?assignees=&labels=Platform%3A+Android%2CType%3A+Bug&template=bug_report_android.yml).
+Let us know if you have feedback through <Link to="https://github.com/getsentry/sentry-java/issues/new?assignees=&labels=Platform%3A+Android%2CType%3A+Bug&template=bug_report_android.yml">GitHub issues</Link>.
```
Comment (vivianyentran) [inline docs/platforms/android/features/index.mdx:16]:
```suggestion
Let us know if you have feedback by filing a <Link to="https://github.com/getsentry/sentry-java/issues/new?assignees=&labels=Platform%3A+Android%2CType%3A+Bug&template=bug_report_android.yml">GitHub issue</Link>.
```

File: docs/platforms/android/features/index.mdx [hunk 2]
```diff
@@ -13,42 +13,42 @@ a set of experimental features to give you more details about app start performa
...
 
 </Expandable>
 
 - Native Development Kit (NDK), the set of tools that that allows you to use C and C++ with Android.
```
Comment (vivianyentran) [inline docs/platforms/android/features/index.mdx:20]:
```suggestion
- Native Development Kit (NDK), the set of tools that allows you to use C and C++ with Android.
```

File: docs/platforms/react-native/features/index.mdx [hunk 3]
```diff
@@ -6,28 +6,29 @@ description: "Learn about the features of Sentry's React Native SDK."
 
 Sentry's React Native SDK enables automatic reporting of errors and exceptions, and identifies performance issues in your application. The below, is a list of features that are available as part of this SDK.
 
-- Automatic Native Crash [Error Tracking](/product/issues/) (using both `sentry-cocoa` & `sentry-android`).
+- Automatic Native Crash <Link to="/product/issues/">Error Tracking</Link> (using both `sentry-cocoa` & `sentry-android`).
+- Automatic detection of <Link to="/platforms/android/configuration/app-not-respond">Application Not Responding (ANR)</Link> on Android and <Link to="/platforms/apple/guides/ios/configuration/app-hangs/">App Hangs</Link> on iOS
```
Comment (vivianyentran) [inline docs/platforms/react-native/features/index.mdx:10]:
```suggestion
- Automatic detection of <Link to="/platforms/android/configuration/app-not-respond">Application Not Responding (ANR)</Link> on Android and <Link to="/platforms/apple/guides/ios/configuration/app-hangs/">App Hangs</Link> on iOS.
```

File: docs/platforms/flutter/features/index.mdx [hunk 4]
```diff
@@ -8,32 +8,33 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
 
 **Features:**
 
-- Under the hood the SDK relies on [Sentry's Dart SDK](/platforms/dart/):
+- Under the hood the SDK relies on <Link to="/platforms/dart">Sentry's Dart SDK</Link>:
     - The minimum required version is `2.17.0` and Flutter SDK version is `3.0.0`.
-    - This SDK includes all the [Features of Sentry's Dart SDK](/platforms/dart/features/)
-- Automatic native crash error tracking (using both [Android](/platforms/android/) and [iOS](/platforms/apple/)), including:
+    - This SDK includes all the <Link to="/platforms/dart/features">Features of Sentry's Dart SDK</Link>
```
Comment (vivianyentran) [inline docs/platforms/flutter/features/index.mdx:13]:
```suggestion
    - This SDK includes all the <Link to="/platforms/dart/features">Features of Sentry's Dart SDK</Link>.
```

File: docs/platforms/flutter/features/index.mdx [hunk 5]
```diff
@@ -8,32 +8,33 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
+- Automatic native crash error tracking (using both <Link to="/platforms/android">Android</Link> and <Link to="/platforms/apple/guides/ios">iOS</Link>), including:
     - Java, Kotlin, C, and C++ code for Android.
     - ObjC, Swift, and C for iOS.
+- Automatic detection of <Link to="/platforms/android/configuration/app-not-respond">Application Not Responding (ANR)</Link> on Android and <Link to="/platforms/apple/guides/ios/configuration/app-hangs/">App Hangs</Link> on iOS
```
Comment (vivianyentran) [inline docs/platforms/flutter/features/index.mdx:17]:
```suggestion
- Automatic detection of <Link to="/platforms/android/configuration/app-not-respond">Application Not Responding (ANR)</Link> on Android and <Link to="/platforms/apple/guides/ios/configuration/app-hangs/">App Hangs</Link> on iOS.
```

File: docs/platforms/flutter/features/index.mdx [hunk 6]
```diff
@@ -8,32 +8,33 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
+    - as well as `http` with the <Link to="/platforms/dart/enriching-events/breadcrumbs/#automatic-breadcrumbs">Dart SDK</Link>.
+- Integrations for sqflite, routing and more. For a complete list, see <PlatformLink to="/integrations">integrations</PlatformLink>.
+- <Link to="/product/releases/health">Release Health</Link> tracks crash free users and sessions.
+- <PlatformLink to="/enriching-events/attachments">Attachments</PlatformLink> that can enrich your event by storing additional files, such as config or log files.
+- <PlatformLink to="/tracing">Tracing</PlatformLink> that can track:
+  - <PlatformLink to="/integrations/app-start-instrumentation">App start time</PlatformLink>
+  - <PlatformLink to="/integrations/routing-instrumentation/#time-to-initial-display">Time to Initial Display and Time to Full Display</PlatformLink>
+  - <PlatformLink to="/integrations/slow-and-frozen-frames-instrumentation">Slow and Frozen Frames</PlatformLink>
+  - <PlatformLink to="/integrations/user-interaction-instrumentation">User Interaction</PlatformLink> which include clicks, long clicks, taps and so on.
```
Comment (vivianyentran) [inline docs/platforms/flutter/features/index.mdx:31]:
```suggestion
  - <PlatformLink to="/integrations/app-start-instrumentation">App start time</PlatformLink>.
  - <PlatformLink to="/integrations/routing-instrumentation/#time-to-initial-display">Time to Initial Display and Time to Full Display</PlatformLink>.
  - <PlatformLink to="/integrations/slow-and-frozen-frames-instrumentation">Slow and Frozen Frames</PlatformLink>.
  - <PlatformLink to="/integrations/user-interaction-instrumentation">User Interaction</PlatformLink> which include clicks, long clicks, taps and so on.
```

**Review by vivianyentran — APPROVED** [review]
Thanks for updating the links too!

File: docs/platforms/react-native/features/index.mdx [hunk 7]
```diff
@@ -6,28 +6,29 @@ description: "Learn about the features of Sentry's React Native SDK."
 
 Sentry's React Native SDK enables automatic reporting of errors and exceptions, and identifies performance issues in your application. The below, is a list of features that are available as part of this SDK.
 
-- Automatic Native Crash [Error Tracking](/product/issues/) (using both `sentry-cocoa` & `sentry-android`).
+- Automatic Native Crash <Link to="/product/issues/">Error Tracking</Link> (using both `sentry-cocoa` & `sentry-android`).
+- Automatic detection of <Link to="/platforms/android/configuration/app-not-respond">Application Not Responding (ANR)</Link> on Android and <Link to="/platforms/apple/guides/ios/configuration/app-hangs/">App Hangs</Link> on iOS.
```
Comment (kahest) [inline docs/platforms/react-native/features/index.mdx:10]: @krystofwoldrich this is new :)

File: docs/platforms/flutter/features/index.mdx [hunk 8]
```diff
@@ -8,32 +8,33 @@ Sentry's Flutter SDK enables automatic reporting of errors and exceptions, and i
...
-- Under the hood the SDK relies on [Sentry's Dart SDK](/platforms/dart/):
+- Under the hood the SDK relies on <Link to="/platforms/dart">Sentry's Dart SDK</Link>:
     - The minimum required version is `2.17.0` and Flutter SDK version is `3.0.0`.
-    - This SDK includes all the [Features of Sentry's Dart SDK](/platforms/dart/features/)
-- Automatic native crash error tracking (using both [Android](/platforms/android/) and [iOS](/platforms/apple/)), including:
+    - This SDK includes all the <Link to="/platforms/dart/features">Features of Sentry's Dart SDK</Link>.
+- Automatic native crash error tracking (using both <Link to="/platforms/android">Android</Link> and <Link to="/platforms/apple/guides/ios">iOS</Link>), including:
     - Java, Kotlin, C, and C++ code for Android.
     - ObjC, Swift, and C for iOS.
+- Automatic detection of <Link to="/platforms/android/configuration/app-not-respond">Application Not Responding (ANR)</Link> on Android and <Link to="/platforms/apple/guides/ios/configuration/app-hangs/">App Hangs</Link> on iOS.
```
Comment (kahest) [inline docs/platforms/flutter/features/index.mdx:17]: @buenaflor this is new :)

Comment (buenaflor) [reply docs/platforms/flutter/features/index.mdx:17]: yup, I think it makes sense

**Pull request merged**